docker-compose.yml

```diff
@@ -47,15 +47,90 @@ services:
       n1l1tos1:
         ipv4_address: 10.168.1.5
       n2htoleaf1:
-      n3htoleafbn1:
-      n5h1tol1r:
       n6h1tol1y:
     command: |
       sh -x -c 'touch /etc/frr/vtysh.conf && \
             sed -i "s/bgpd=no/bgpd=yes/g" /etc/frr/daemons && \
             sed -i "s/127.0.0.1/0.0.0.0/g" /etc/frr/daemons && \
             ip link add name lo0 type dummy && \
             ifconfig lo0 10.0.0.2 netmask 255.255.255.255 up && \
+
+            ip link add br-tenant type bridge vlan_default_pvid 0 vlan_filtering 1 vlan_protocol 802.1Q && \
+            ip link set br-tenant up && \
+
+            ip link add blue type vrf table 1000 && \
+            ip link set blue up && \
+            ip link add green type vrf table 1001 && \
+            ip link set green up && \
+            ip link add yellow type vrf table 1002 && \
+            ip link set yellow up && \
+
+            ip link add br100 addr aa:bb:cc:00:00:02 type bridge && \
+            ip link set br100 master green up && \
+            ip link add vni100 type vxlan local 10.0.0.2 dstport 4789 id 100 nolearning && \
+            ip link set vni100 master br100 up && \
+
+            ip link add br101 addr aa:bb:cc:00:00:03 type bridge && \
+            ip link set br101 master yellow up && \
+            ip link add vni101 type vxlan local 10.0.0.2 dstport 4789 id 101 nolearning && \
+            ip link set vni101 master br101 up && \
+
+            bridge vlan add dev br-tenant vid 10 self && \
+            ip link add vni10 type vxlan id 10 local 10.0.0.2 dstport 4789 nolearning && \
+            ip link set vni10 master br-tenant up && \
+            bridge vlan add dev vni10 vid 10 pvid untagged && \
+            bridge link set dev vni10 neigh_suppress on learning off && \
```

Comment:
This is the command that I have

bridge link set dev vni10 neigh_suppress on

Reply:
When the BGP EVPN is enabled, we expect no learning on VXLAN tunnel interface as it is programmed by BGP EVPN module.

Reply:
Yes that makes sense. I am not sure why we do not have that. Ok leave it as it is for now and we see